Sources/.swiftlint.yml

```diff
@@ -43,6 +43,7 @@ function_body_length:
 
 identifier_name:
   excluded: [r, g, b, a, x, y, z, dx, dy, dz, i, j, k, id, op, or, me, at, to, in]
+  allowed_symbols: ["_"]
```

Comment:
I still don't think we need this.

Reply:
We talked about it already :) https://getstream.slack.com/archives/CRB9P6L03/p1588260809278800?thread_ts=1588260684.278300&cid=CRB9P6L03